### PR TITLE
JAPI-482 Automate HPCC4J tests vs containerized version of HPCC

### PR DESCRIPTION
**Pull request by jpmcmu**
- Modified httpsUnitTests to use K8s / Helm
- Removed uncessary remoteUnitTests
- Improved test data generation workflow

Signed-off-by: James McMullan James.McMullan@lexisnexis.com

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [X] I have created a corresponding JIRA ticket for this submission
- [X] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [X] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [X] The change has been fully tested:
  - [ ] This change does not cause any existing JUnits to fail.
  - [ ] I have include JUnit coverage to test this change
  - [ ] I have performed system test and covered possible regressions and side effects.
- [X] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] This change fixes the problem, not just the symptom

### CODE REVIEW

File: wsclient/src/test/java/org/hpccsystems/ws/client/WSSQLClientTest.java [hunk 1]
```diff
@@ -41,8 +41,10 @@ HPCC SYSTEMS software Copyright (C) 2019 HPCC Systems®.
 import org.hpccsystems.ws.client.wrappers.gen.wssql.QuerySetQueries_type0Wrapper;
 import org.hpccsystems.ws.client.wrappers.gen.wssql.QuerySignatureWrapper;
 import org.junit.Assert;
+import org.junit.Assume;
 import org.junit.FixMethodOrder;
 import org.junit.Test;
+import org.junit.Ignore;
```
Comment (rpastrana) [inline wsclient/src/test/java/org/hpccsystems/ws/client/WSSQLClientTest.java:47]: do we still need this import?

File: wsclient/src/test/java/org/hpccsystems/ws/client/WsClientTest.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package org.hpccsystems.ws.client;
 
 import org.hpccsystems.ws.client.platform.test.BaseRemoteTest;
+
 import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Ignore;
```
Comment (rpastrana) [inline wsclient/src/test/java/org/hpccsystems/ws/client/WsClientTest.java:7]: most of these Ignore imports might be left-overs from previous commit

**Review by rpastrana — COMMENTED** [review]
@jpmcmu let's clean up the imports, and you mentioned a potential fix for the dfsclient test

**Comment by jpmcmu** [comment]
@rpastrana if this looks good I will go ahead and squash

**Pull request merged**